.changeset/real-falcons-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"@typescript/ata": minor
```

Comment:
I almost wonder if it should just be a patch, since it should have worked before?